packages/commons/src/TypedEventEmitter/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export * from './TypedEventEmitter';
```

Comment:
Suggestion: How about also exporting it at the root of the `commons` package? 
So that we could import it directly with 

```
import {TypedEventEmitter} from '@wireapp/commons'
```

Reply:
added with https://github.com/wireapp/wire-web-packages/pull/5551/commits/2df53eae5d7b27cb67d7984a4ca403951d80d84d